src/T2DMap.cpp

```diff
@@ -2090,10 +2090,12 @@ void T2DMap::paintEvent(QPaintEvent* e)
             }
         }
 
+#ifdef DEBUG
```

Comment:
That isn't the correct check for a debug build IIRC (it gets used in the `cTelnet` class currently but that is because it is `#define`d at the top of the file - don't you want `if (mudlet::scmIsDevelopmentVersion) {` instead?